style.css

```diff
@@ -63,16 +68,166 @@
   font-size: 16px;
   line-height: 24px;
   text-align: left;
-  padding-left: 5%;
   padding-top: 5%;
+  background: transparent;
 }
 
 .intro-section__social {
   display: flex;
   flex-direction: row;
-  margin: 4%;
+  padding: 4% 4% 4% 0;
+  background: transparent;
 }
 
-.intro-section__social > * {
+.intro-section__social > li {
   padding: 2%;
+  background: transparent;
+}
+
+.project__section {
+  display: flex;
+  flex-direction: column;
+  padding: 5rem 0 5rem 0;
+}
```

Comment:
- [ ] Great work applying styles to your projects section 💯. However, please kindly ensure that you use both display grid and display flex to build this section as requested in the requirement below. 👇🏽

    >In order to lay out the elements in "Works" section you should use CSS Grid and Flexbox combined (it is mandatory that you use both). Use CSS Grid in the main container that contains all of the cards and use Flexbox to lay out an individual card.